keybase/platform_test.go

```diff
@@ -21,12 +21,6 @@ func (c testConfigPlatform) promptProgram() (command.Program, error) {
 	if programPath == "" {
 		programPath = filepath.Join(os.Getenv("GOPATH"), "bin", "test")
 	}
-	if len(args) == 0 {
```

Comment:
Moved this default arg into the 1 test that was using it (should be with the test, not a default).